Xamarin.Forms.Platform.Android/Platform.cs

```diff
@@ -1109,7 +1078,8 @@ internal static int GenerateViewId()
 		internal static readonly BindableProperty PageContextProperty = 
 			BindableProperty.CreateAttached("PageContext", typeof(Context), typeof(Platform), null);
 
-		internal Platform(Context context) : this(context, false)
+		// This version of the constructor is for the Previewer
+		internal Platform(Context context) : this((Activity)context, false)
```

Comment:
This is still going to break the same way than before by producing an `InvalidCastException`

Reply:
Which is by the way exactly one of the issue that @davidortinau shows in his log